sendgrid/base_interface.py

```diff
@@ -60,3 +63,31 @@ def send(self, message):
             message = message.get()
 
         return self.client.mail.send.post(request_body=message)
+
+    def set_host(self,host):
+        self.host = host
+        self.client = python_http_client.Client(
+            host=self.host,
+            request_headers=self._default_headers,
+            version=3)
+
+    def set_data_residency(self,region):
+        """
+        * Client libraries contain setters for specifying region/edge.
+        * This allows support global and eu regions only. This set will likely expand in the future.
+        * Global should be the default
+        * Global region means the message should be sent through:
+        * HTTP: api.sendgrid.com
+        * EU region means the message should be sent through:
+        * HTTP: api.eu.sendgrid.com
```

Comment:
i think these comments can be worded differently. It should say what it is doing.
something like:
```suggestion
        * Global is the default residency (or region)
        * Global region means the message will be sent through https://api.sendgrid.com
        * EU region means the message will be sent through https://api.eu.sendgrid.com
```